src/Storages/NATS/NATSProducer.h

```diff
@@ -16,7 +17,10 @@ class NATSProducer : public AsynchronousMessageProducer
     NATSProducer(
         const NATSConfiguration & configuration_,
         const String & subject_,
-        std::atomic<bool> & shutdown_called_,
+        std::
+        
+        
+        c<bool> & shutdown_called_,
```

Comment:
I have no idea how it ended up like this, but there's no doubt why the style check complained about this :sweat_smile: 

https://github.com/ClickHouse/ClickHouse/actions/runs/13144084166/job/36678089178?pr=75299